cypress/support/commands.js

```diff
@@ -74,3 +74,19 @@ Cypress.Commands.add(
     });
   }
 );
+
+Cypress.Commands.add("registerUser", (handle = "nope", id = "123abcd.git") => {
+  client.mutate({
+    variables: {
+      handle: handle,
+      id: id
+    },
+    mutation: gql`
+      mutation RegisterUser($handle: String, $id: String) {
+        registerUser(handle: $handle, id: $id) {
+          handle
+        }
+      }
+    `
```

Comment:
The query is not correct here. handle and id should be `ID` and required and just `handle` can't be returned as such. See the query in `UserRegistration.svelte`, copying it over here makes the test green.